lib/tests/it/main.rs

```diff
@@ -67,30 +42,40 @@ async fn test_tar_import_empty() -> Result<()> {
 
 #[tokio::test]
 async fn test_tar_export_reproducible() -> Result<()> {
-    let fixture = Fixture::new()?;
-    let (_, rev) = fixture
-        .srcrepo
-        .read_commit(fixture.testref(), gio::NONE_CANCELLABLE)?;
-    let export1 = {
-        let mut h = openssl::hash::Hasher::new(openssl::hash::MessageDigest::sha256())?;
-        ostree_ext::tar::export_commit(&fixture.srcrepo, rev.as_str(), &mut h, None)?;
-        h.finish()?
-    };
-    // Artificial delay to flush out mtimes (one second granularity baseline, plus another 100ms for good measure).
-    std::thread::sleep(std::time::Duration::from_millis(1100));
-    let export2 = {
-        let mut h = openssl::hash::Hasher::new(openssl::hash::MessageDigest::sha256())?;
-        ostree_ext::tar::export_commit(&fixture.srcrepo, rev.as_str(), &mut h, None)?;
-        h.finish()?
-    };
-    assert_eq!(*export1, *export2);
+    for fixture in [Fixture::new()?, Fixture::new_v1()?] {
+        let (_, rev) = fixture
+            .srcrepo
+            .read_commit(fixture.testref(), gio::NONE_CANCELLABLE)?;
+        let export1 = {
+            let mut h = openssl::hash::Hasher::new(openssl::hash::MessageDigest::sha256())?;
+            ostree_ext::tar::export_commit(&fixture.srcrepo, rev.as_str(), &mut h, None)?;
+            h.finish()?
+        };
+        // Artificial delay to flush out mtimes (one second granularity baseline, plus another 100ms for good measure).
+        std::thread::sleep(std::time::Duration::from_millis(1100));
+        let export2 = {
+            let mut h = openssl::hash::Hasher::new(openssl::hash::MessageDigest::sha256())?;
+            ostree_ext::tar::export_commit(&fixture.srcrepo, rev.as_str(), &mut h, None)?;
+            h.finish()?
+        };
+        assert_eq!(*export1, *export2);
```

Comment:
Two alternatives options, just as suggestions for your consideration:
 1. Hardcode a list of digests from the current baseline, to ensure the exports do not unexpectedly diverge over time
 2. Same as 1, but also also committing the fixtures (which are programmatically generated by the testsuite) for easier debugging on regressions

Reply:
One problem with hardcoding the digests of the tar stream is that it can change if the bitstream from [tar-rs](https://github.com/alexcrichton/tar-rs) changes.

Now, on one hand we need to be aware of this, because bitstream generation changes there will actually be very noticeable after we implement https://github.com/ostreedev/ostree-rs-ext/pull/123 because it will force clients to redownload *all* layers.